migration/sql-files/068-spaces_identities_fk.sql

```diff
@@ -0,0 +1,8 @@
+-- add a foreign key from spaces.owner_id to identities.id
+alter table spaces add constraint spaces_owner_id_identities_id_fk foreign key (owner_id) REFERENCES identities (id);
+-- add index on spaces.name
+create index idx_spaces_name on spaces (lower(name));
+-- add index on identities.username
+drop index idx_identities_username;
+create index idx_identities_username on identities (lower(username));
```

Comment:
Have you verified/changed whatever is using that index today to now use lower(username) = lower(?) so they still hit the same index?